### ISSUE TITLE
ES not working in scirius ,the status light is gray  

### ISSUE THREAD

**Issue by nyb123456nyb** [StamusNetworks/scirius#182]
HI gays:

   I am from china , I have an issue which i cannit solved ,thanks for help!
my scirius is   SCIRIUS_VERSION="3.2.0,ES verison is Version: 6.6.2. 
![image](https://user-images.githubusercontent.com/21351974/61528173-abec3380-aa50-11e9-8e9b-bb81b74b164d.png)
my scirius setting.py is :
![image](https://user-images.githubusercontent.com/21351974/61528282-eeae0b80-aa50-11e9-9c7f-aa63a18b4e8c.png)
the error info  in elasticsearch.log  is 

![image](https://user-images.githubusercontent.com/21351974/61528404-40569600-aa51-11e9-8d0e-04436278af8e.png)

**Comment by pevma** [comment]
Is that SELKS 5.0 ?
It seem like an ``auth`` err - have you done any changes to the default config - i see `http auth` adjustments ?

**Comment by nyb123456nyb** [comment]
ELK is 6.6.2  ,suricata is 4.1.0, I changed default config in scirius setting.py  with add "http auth".

**Comment by nyb123456nyb** [comment]
my ELK installed on 10.104.116.212, my scirius and suricata   installed on 192.168.13.128.  I don't know if it has anything to do with this. thanks!

**Comment by pevma** [comment]
Why did you do the change , what was needed?  In SELKS authentication is done via Scirius by default and it works that way - hence my question  - is it a diff auth mechanism you are using or ?

**Comment by nyb123456nyb** [comment]
Because my es is configured with authentication，I find nothing with  authentication in scirius setting.py.
When starting with the default configuration。I got error is :：
![image](https://user-images.githubusercontent.com/21351974/61782444-a3bd3b00-ae38-11e9-8939-a78ae9216030.png)
![image](https://user-images.githubusercontent.com/21351974/61782474-ae77d000-ae38-11e9-8277-5591b150e564.png)

**Comment by nyb123456nyb** [comment]
Now I am talking about my use environment. I have distributed several suricata distributed at various network boundaries, and then ES and kibana are deployed at 10.104.116.212 to display the alarm events. Logstash is deployed at 10.3.4.79, and the event is played. Into the ES, since the previous rules were managed by oinkmaster, now I want to deploy the scirius management rules and deploy the scirius at 10.104.116.212. Thank you very much for helping

**Comment by pevma** [comment]
What authentication is used on es , do you have Xpack enabled ?

**Comment by nyb123456nyb** [comment]
Yes ,use Xpack on es, the elasticsearch.yml is:
![image](https://user-images.githubusercontent.com/21351974/62201020-23a75000-b3b9-11e9-8b40-9f3068ade0fe.png)

**Comment by pevma** [comment]
Currently Scirius is in `charge` of authentication and uses a proxy so it is not fully compatible yet with Xpack. 
To confirm that - can you disable Xpack security / auth, adjust the settings accordingly in `/etc/scirius/local_settings.py` and restart the machine and try again ?

**Comment by nyb123456nyb** [comment]
Yes ,disable Xpack on ES ,then ok .  but  scirius still no eve data
![image](https://user-images.githubusercontent.com/21351974/62270239-1bf2b480-b468-11e9-8f27-f7e241c7bcf5.png)
and kibana still error:
![image](https://user-images.githubusercontent.com/21351974/62270263-2c0a9400-b468-11e9-8791-11a2b4731007.png)
![image](https://user-images.githubusercontent.com/21351974/62270275-3462cf00-b468-11e9-9ee5-6c0890f0c952.png)

**Comment by pevma** [comment]
It seems it could be a proxy issue - prohibiting the page display

**Comment by nyb123456nyb** [comment]
But I didn't set up the proxy, it's really strange,  I  don't know how to solve it. 😂😭😭😭😭😭😭

**Comment by pevma** [comment]
I think you could try in a test setup - fresh install without xpack enabled - to see if you will get the same err?
